### ISSUE TITLE
Improve Vacuum & Bunny Collisions

### ISSUE THREAD

**Issue by hawkins** [VideoGameDevClub/VacuumCleanSupreme#2]
Require bunny to be at or below vacuum, not simply within some range.

Additionally, consider Mario-esque features where Bunny could jump on top of a Vacuum to stun it or spin it around.

**Comment by Blond-in-Blue** [comment]
Ideally, in this game mode, bunnies won't be able to touch the top of the vacuum cleaner. Being able to jump on top of the vacuum to disable it in a 4 vs 1 scenario would make the bunnies overpowered and open up griefing opportunities, making the game not fun for the player controlling the vacuum.

However, requiring that the vacuum make contact with the bunnies within a certain arc (such as the front of the vacuum, or whichever side has forward velocity) to suck them up would make sense.

**Comment by hawkins** [comment]
Good point. I'll rewrite the original issue to indicate for use in other potential game modes.
